package.json

```diff
@@ -42,46 +49,58 @@
   },
   "homepage": "https://github.com/carloscuesta/react-native-error-boundary",
   "peerDependencies": {
+    "@types/react-native": ">=0.57.7",
     "react": ">=16.6.0",
     "react-native": ">=0.57.7"
   },
+  "peerDependenciesMeta": {
+    "@types/react-native": {
+      "optional": true
+    }
+  },
```

Comment:
More info about this here:

https://reactnative.dev/blog/2023/01/03/typescript-first#library-maintainer

> <img width="823" alt="Screenshot 2023-01-08 at 22 26 16" src="https://user-images.githubusercontent.com/7629661/211219914-8c08201f-d4f1-4be7-ad96-6e0611064d3f.png">